modules/javafx.graphics/src/main/java/javafx/scene/text/TextFlow.java

```diff
@@ -199,7 +199,7 @@ public final HitInfo hitTest(javafx.geometry.Point2D point) {
             TextLayout layout = getTextLayout();
             double x = point.getX();
             double y = point.getY();
-            TextLayout.Hit h = layout.getHitInfo((float)x, (float)y, null, 0, 0);
+            TextLayout.Hit h = layout.getHitInfo((float)x, (float)y, null, 0, 0, true);
```

Comment:
The `null` and `true` here are not needed IMHO (what would even happen when they conflict, `null` + `false`, or non-`null` + `true`?)

Reply:
Removed last parameter